src/metabase/actions/error.clj

```diff
@@ -0,0 +1,17 @@
+(ns metabase.actions.error)
```

Comment:
_leaving comment in an unrelated place_

I tried to execute an action with parameters that would violate 2 different FK constraints, but I'm only getting error for one of them:

![image](https://github.com/metabase/metabase/assets/6830683/4020069b-ebef-4c78-8f0a-250eaecd3f8f)

----

If I try to violate constraints separately, I correctly receive the detailed errors:

![image](https://github.com/metabase/metabase/assets/6830683/0c41ec4e-8fe1-4662-9eca-ee0594e63237)